.gitignore

```diff
@@ -20,3 +20,6 @@
 tags
 
 .idea
+
+/html/*
+/latex/*
```

Comment:
I'm not sure. I was looking at `/examples/baseexample.cfg`, and the output directory is `OUTPUT_DIRECTORY   = ..`, so I would expect the output to the root of the repo `/`

Also, if you want to keep this change, do you want me to create another PR for that?

Reply:
I now understand how you got the `html` and `latex`, the examples are not intended for running them standalone, but as part of the documentation generation in the build process (`make docs` and `make docs_chm`), so other people might indeed also run into this problem.

I think this would be better and cleaner to have a separate PR for this.